vector/src/main/java/im/vector/app/core/pushers/PushersManager.kt

```diff
@@ -70,6 +70,8 @@ class PushersManager @Inject constructor(
             appNameProvider.getAppName(),
             activeSessionHolder.getActiveSession().sessionParams.deviceId ?: "MOBILE",
             gateway,
+            enabled = true,
+            deviceId = activeSessionHolder.getActiveSession().sessionParams.deviceId ?: "MOBILE",
```

Comment:
@ouchadam there is some obvious duplication here with the above field `deviceDisplayName`. It looks like it's not given us any problems using a "device id" as the display name, but the api spec expects this display name to be something more readable like "iPhone 9". I don't see that we currently get this data anywhere. What are your thoughts?

Reply:
sounds like a good change to make (assuming iOS is doing the same) 

at the moment the app doesn't display the `deviceDisplayName` anywhere from what I can tell~

![Screenshot_20220923_085600](https://user-images.githubusercontent.com/1848238/191915619-6e1500a7-6503-471d-a203-1062e5787095.png)

*pusher deleted after taking the screenshot*

Reply:
I opted to use `android.os.Build.MODEL` instead for `deviceDisplayName`. Will catch up with @gileluard on the same for parity, but I believe this is the right approach